node_modules/oae-search/lib/searches/email.js

```diff
@@ -63,3 +64,21 @@ module.exports = function(ctx, opts, callback) {
     var filter = SearchUtil.filterAnd(filterResources, filterInteractingTenants);
     return callback(null, SearchUtil.createQuery(query, filter, queryOpts));
 };
+
+/**
+ * Post process a search result set
```

Comment:
Since it's implied that it's post processing a search result set, can this doc be more specific to what it's doing (i.e., indicating on the search result the tenant to which an email belongs)?